packages/components/inputs/date-input/src/date-input.tsx

```diff
@@ -279,7 +284,7 @@ const DateInput = (props: TDateInput) => {
                       clearSelection();
                     }
                     // ArrowDown
-                    if (event.keyCode === 40) {
+                    if (event.key === 'ArrowDown') {
```

Comment:
I was wondering about our browsers version support policy as I read this [in MDN](https://devdocs.io/dom/keyboardevent/key/key_values):
![image](https://user-images.githubusercontent.com/97907068/158221543-fd7d7541-4dd0-4133-8f9e-c459d17884c0.png)

Do we publicly state which browsers & versions we support?

Reply:
We have some info here

https://docs.commercetools.com/custom-applications/help-needed #browser-support

So it should be fine, as those mentioned there are pretty old versions